ios/RNMagicScript/components/UiNodes/UiSliderNode.swift

```diff
@@ -141,31 +144,40 @@ import SceneKit
             self.height = height
         }
 
-        if let min = Convert.toCGFloat(props["min"]) {
+        if let max = Convert.toCGFloat(props["max"]),
```

Comment:
Can you store the result of `Convert.toCGFloat()` in a local variable so that it is calculated once?